apps/store-admin/controllers/index-router.jag

```diff
@@ -0,0 +1,13 @@
+<%
```

Comment:
None of the jag files and js files are having the license header, please add those.